exporters/otlp/otlpmetric/otlpmetricgrpc/client.go

```diff
@@ -196,9 +198,12 @@ func (c *client) UploadMetrics(ctx context.Context, protoMetrics *metricpb.Resou
 	defer cancel()
 
 	return c.requestFunc(ctx, func(iCtx context.Context) error {
-		_, err := c.msc.Export(iCtx, &colmetricpb.ExportMetricsServiceRequest{
+		resp, err := c.msc.Export(iCtx, &colmetricpb.ExportMetricsServiceRequest{
 			ResourceMetrics: []*metricpb.ResourceMetrics{protoMetrics},
 		})
+		if resp != nil && resp.PartialSuccess != nil {
+			otel.Handle(otlpinternal.PartialSuccessToError("data points", resp.PartialSuccess.RejectedDataPoints, resp.PartialSuccess.ErrorMessage))
```

Comment:
```suggestion
			otel.Handle(otlpinternal.PartialSuccessToError("metric data points", resp.PartialSuccess.RejectedDataPoints, resp.PartialSuccess.ErrorMessage))
```
It wouldn't be obvious to me that a data point is necessarily a metric data point.  This makes it clearer.